apps/src/lib/levelbuilder/code-docs-editor/NewProgrammingEnvironmentForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import RailsAuthenticityToken from '@cdo/apps/lib/util/RailsAuthenticityToken';
+import HelpTip from '@cdo/apps/lib/ui/HelpTip';
+
+export default function NewProgrammingEnvironmentForm() {
+  return (
+    <form action="/programming_environments" method="post">
+      <RailsAuthenticityToken />
+      <h1>New Programming Environment</h1>
+      <h2>
+        This feature is in development. Please continue to use curriculum
+        builder to create code documentation.
+      </h2>
+      <label>
+        Programming Environment Slug
+        <input name="name" style={styles.inputStyle} required />
+        <HelpTip>
+          <p>
+            The programming environment slug is used in URLs and cannot be
+            updated once set. A slug can only contain lowercase letters,
+            numbers, underscores, and dashes. Likely, this should be the same at
```

Comment:
Should we allow both underscores and dashes or just pick one?

Reply:
I was debating that and feeling a bit lazy so I left both. But now I'm thinking we should pick dashes and remove underscores. If we need them later, it'll be easy to allow them.

Reply:
That sounds good me